__tests__/index.test.js

```diff
@@ -42,10 +35,73 @@ describe("nc news", () => {
                 .then(({body}) => {
                     expect(body.endpoint).toMatchObject({
                         "GET /api": expect.any(Object), "GET /api/topics": expect.any(Object), "GET /api/articles": expect.any(Object)
+                    })     
+                })
+            })
+        })
+        describe("/api/articles", ()=> {
+            test("200: serves an array of all articles", () => {
+                return request(app)
+                    .get("/api/articles")
+                    .expect(200)
+                    .then(({body}) => {
+                      expect(body.articles.length >= 1).toBe(true)
+                      body.articles.forEach((article) => {
+                        expect(article).toMatchObject({
+                            article_id: expect.any(Number), title: expect.any(String), topic: expect.any(String), author: expect.any(String),
+                            body: expect.any(String), created_at: expect.any(String), votes: expect.any(Number), article_img_url: expect.any(String)
                     })
-                      
+                })          
+            })
+        })
+    })
+     describe("/api/articles", ()=> {
+            test("200: serves an array of all articles", () => {
+                return request(app)
+                    .get("/api/articles")
+                    .expect(200)
+                    .then(({body}) => {
+                      expect(body.articles.length >= 1).toBe(true)
+                      body.articles.forEach((article) => {
+                        expect(article).toMatchObject({
+                            article_id: expect.any(Number), title: expect.any(String), topic: expect.any(String), author: expect.any(String),
+                            body: expect.any(String), created_at: expect.any(String), votes: expect.any(Number), article_img_url: expect.any(String)
                     })
-                })
+                })          
+            })
+        })
+    })
+    describe("/api/articles/:articles_id", ()=> {
+        test("200: sends a single article to the client based by ID", () => {
+            return request(app)
+                .get("/api/articles/1")
+                .expect(200)
+                .then(({body}) => {
+                  expect(body.article.article_id).toBe(1);
+                  expect(body.article.title).toBe('Living in the shadow of a great man');
```

Comment:
For your 200 test - to allow for later extension of the endpoint, the test should use a flexible matcher eg expect.objectContaining or .toMatchObject. We only need to specifically check the value of the article_id (to ensure it is the correct article)